xskillscore/core/comparative.py

```diff
@@ -1,10 +1,14 @@
+import inspect
 import warnings
+from typing import List, Mapping, Optional, Tuple, Union
 
 import numpy as np
 import scipy.stats as st
 import xarray as xr
 
-from .deterministic import mae, pearson_r
+from . import deterministic as dm
```

Comment:
I know it adds a few more lines but I prefer the imports e.g. https://github.com/xarray-contrib/xskillscore/blob/main/xskillscore/core/accessor.py#L3

Reply:
I like his implementation

Reply:
given that i've never touched comparative and don't have any plans to i'll follow your suggestion here.